### PR TITLE
Register environments with entrypoint in setup.py

### PR DESCRIPTION
**Pull request by rodrigodelazcano**
# Description
Specify an entrypoint in the setup.py to create a plugin for the gym_minigrid environments under the name space `gym.envs`. This will allow to initialize environments directly with the environment name id without having to import `gym_minigrid` or specifying the module name like `gym.make('gym_minigrid:env_id_name').

**Before this PR**
- Unused import:
```
import gym_minigrid
import gym

env = gym.make('MiniGrid-PutNear-6x6-N2-v0')
```
- Long environment name:
```
import gym

env = gym.make('gym_minigrid:MiniGrid-PutNear-6x6-N2-v0')
```

**After this PR**
```
import gym

env = gym.make('MiniGrid-PutNear-6x6-N2-v0')
```
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Screenshots
Please attach before and after screenshots of the change if applicable.

# Checklist:

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: gym_minigrid/minigrid.py [hunk 1]
```diff
@@ -751,7 +751,11 @@ def reset(self, *, seed=None, return_info=False, options=None):
 
         # Return first observation
         obs = self.gen_obs()
-        return obs
+
```
Comment (pseudo-rnd-thoughts) [inline gym_minigrid/minigrid.py:754]: This should be part of a separate PR

Comment (rodrigodelazcano) [reply gym_minigrid/minigrid.py:754]: Yes, sorry I'll remove it

**Comment by pseudo-rnd-thoughts** [comment]
Should we not remove the register.py file with this change?

**Review by pseudo-rnd-thoughts — APPROVED** [review]
LGTM
